packages/framework/src/Model/Product/Pricing/ProductManualInputPriceFacade.php

```diff
@@ -41,6 +41,5 @@ public function refresh(Product $product, PricingGroup $pricingGroup, ?Money $in
             $manualInputPrice->setInputPrice($inputPrice);
         }
         $this->em->persist($manualInputPrice);
-        $this->em->flush();
```

Comment:
I feel a little bit concerned about removing `flush` from the public facade methods. Basically, it means you need to keep it in mind when using the method and call it "manually" wherever you use it (which can be already seen - you had to add the `flush` call into `CartWatcherTest::testGetModifiedPriceItemsAndUpdatePrices()`

Reply:
I can live with that in the protected methods (e.g. you removed it from `GoogleProductDomainFacade::saveGoogleProductDomain` and I would be ok with that if the visibility was changed to protected -  it is not used outside the class)

Reply:
...the same applies to the other `*ProductDomainFacade::save*` methods (`HeurekaProductDataFixture` and `ZboziPluginDataFixture` need to be adjusted in such a case)

Reply:
I have changed methods to protected ones